bin/setup-rails

```diff
@@ -23,12 +23,9 @@ FileUtils.chdir APP_ROOT do
   puts "\n== Copying sample files =="
   system!('bin/copy-example-envs')
 
-  puts "\n== Preparing development database =="
-  system! 'bin/rails db:prepare'
-
-  puts "\n== Seeding development database according to feature settings in .env =="
-  system! 'bin/rake db:seed'
-  system! 'bin/rake release:after_build'
+  puts "\n== Creating and seeding development database =="
+  system! 'bin/rails db:setup'
```

Comment:
I think `prepare` is the new preference because it will setup *or* migrate; whereas setup presumes a new database.

Reply:
Ah, I didn't realize was nicely idempotent like that! Done.